files/common/var/lib/delphix-platform/os-migration

```diff
@@ -20,8 +20,8 @@ MIGRATE_CONFIG_LOG="/var/delphix/migration/log"
 PG_REINDEX=/var/delphix/server/db/force-reindex
 
 #
-# The perform-migration flag file is created by dx_execute prior rebooting into
-# Linux.
+# The perform-migration flag file is created by dx_execute prior rebooting
```

Comment:
This isn't really a comment on this PR.. but can you remind me why we chose the current design.. where delphix-migration is always enabled by default, but it only "does stuff" when this file is present.. vs. a design where delphix-migration is disabled by default, and dx_execute would enable it for a migrated appliance? I found it a little strange that on first boot of a non-migrated appliance, delphix-migration was running.

Reply:
The main reason is that we cannot run `systemctl enable delphix-migration` in Illumos prior to rebooting the system.

That said, if all that enable does is create the symlinks from the `Install` section, perhaps we can leverage that instead.